armada/armada.py

```diff
@@ -24,16 +24,14 @@ def __init__(self, args):
 
         self.tiller = Tiller()
 
-
     def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, version, chart, values in known_releases:
+        for chart_name, _, chart, values in known_releases:
```

Comment:
i agree maybe in the future the version is needed hence maybe this ignoring it for now is okay. in the case that we will never needed it. we should create a new issue in order to cover that change. i dont really want to start making code changes in this PR since the main goal of it is to just fix the linting.